From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@AndyHerb__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@squillace & @cynthn - VM pull request

tysonn commented:
@AndyHerb - Due to some refactoring, I can't merge this PR. However, I did make the update in the branch we are using for the refactoring - sorry we couldn't merge it directly.

Commit:
Note updates

v-nagta committed:
USER STORY 1178697 SaaS App Tutorial: Confluence SAML SSO by Microsoft Update

v-nagta committed:
updatedmdliks

v-nagta committed:
updated

v-nagta committed:
updated

ShawnJackson committed:
edit pass: virtual-machine-scale-sets-upgrade-scale-set

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-machine-scale-sets-upgrade-scale-set

ShawnJackson committed:
edit pass: virtual-machine-scale-sets-upgrade-scale-set

cephalin committed:
new cmdlet usage

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-machine-scale-sets-upgrade-scale-set

ShawnJackson committed:
edit pass: virtual-machine-scale-sets-upgrade-scale-set

ShawnJackson committed:
edit pass: virtual-machine-scale-sets-upgrade-scale-set

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

eringreenlee committed:
added service principal alert

added alert for password sync

cherylmc committed:
Update how-to-move-peering.md

eringreenlee committed:
fixed for a more conversational tone

pharring committed:
Update uploader log example and add Web Role sample code

pharring committed:
Update troubleshoot page with new uploader name

pharring committed:
Adopt some Acrolinx suggestions

kgremban committed:
Updating module comp article for issue #5346

kgremban committed:
Update H2s, fix links

gsacavdm committed:
Add entry for documentation-government-k8.md

gsacavdm committed:
Add documentation-government-k8.md

gsacavdm committed:
Fix acrolinx issues in documentation-government-k8

gsacavdm committed:
Few minor fixes on documentation-government-k8

gsacavdm committed:
Fixing gov-k8 links

msjuergent committed:
wa_doc7

msjuergent committed:
wa_doc8

msjuergent committed:
wa_doc9

christiankuhtz committed:
tweak

pharring committed:
Merge remote-tracking branch 'upstream/master'

gsacavdm committed:
Fix gov-k8 cli link

SnehaGunda committed:
Adding VS tools edge topic

gsacavdm committed:
Fix gov-k8 note and warning

SnehaGunda committed:
Updating links

SnehaGunda committed:
Updating table

v-geberr committed:
version concept

eringreenlee committed:
fixed alert text for 105

v-geberr committed:
edits

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0313-concept-versions

v-geberr committed:
toc addition

v-geberr committed:
az cli cognitive services

v-geberr committed:
toc fix

v-geberr committed:
added azure cli and powershell rm to reference

neilpeterson committed:
updated to AKS

v-geberr committed:
edits

SnehaGunda committed:
Updating links

v-geberr committed:
new image

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into mar12_5346

kgremban committed:
Updated with info about routing from leaf devices

SnehaGunda committed:
Adding image

eringreenlee committed:
fixed conversational tone

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into mar12_5346

SnehaGunda committed:
Updating image

v-geberr committed:
added links

SnehaGunda committed:
Fixing acrolinx

v-geberr committed:
fix samples node

CarlRabeler committed:
[sql db] metadata cleanup 1

SnehaGunda committed:
Fixing acrolinx

douglaslMS committed:
Moved new article in TOC per PM request.

kgremban committed:
Add clarifying info about ACR

kgremban committed:
Fixed typos and next steps

v-geberr committed:
utterance limit for import is 10K

v-geberr committed:
glossary spelling fix

cherylmc committed:
Update how-to-move-peering.md

christiankuhtz committed:
tweak

CarlRabeler committed:
[sql db] meta data cleanup 2

douglaslMS committed:
Converted em dash back to en dash for consistency.

GitHubber17 committed:
Merge pull request #35219 from v-geberr/0313-concept-versions

version concept

cherylmc committed:
Update how-to-move-peering.md

PRMerger12 committed:
Merge pull request #35248 from v-geberr/0313-glossary-fix-2

glossary spelling fix

kgremban committed:
Update module-composition.md

PRMerger12 committed:
Merge pull request #35249 from christiankuhtz/patch-75

tweak

neilpeterson committed:
updates

j-martens committed:
Merge branch 'patch-19' of https://github.com/raymondlaghaeian/azure-docs-pr

# Conflicts:
#	articles/machine-learning/preview/tutorial-classifying-iris-part-3.md

GitHubber17 committed:
Merge pull request #35233 from v-geberr/0313-samples-azure-cli

0313 samples azure cli

CarlRabeler committed:
Update sql-database-elastic-database-client-library.md

PRMerger15 committed:
Merge pull request #35246 from v-geberr/0313-boundaries

utterance limit for import is 10K

barbkess committed:
acrolinx fixes

neilpeterson committed:
name updated for consistancy

PRMerger17 committed:
Merge pull request #35252 from j-martens/master

AML: bringing in Raymond's edits

kgremban committed:
Update module-composition.md

PRMerger17 committed:
Merge pull request #35102 from cherylmc/patch-2

Update how-to-move-peering.md

PRMerger18 committed:
Merge pull request #34770 from cephalin/cmdlet

new cmdlet usage

CarlRabeler committed:
[sql db] metadata clean-up 3

GitHubber17 committed:
Merge pull request #35243 from CarlRabeler/meta-1

[sql db] metadata cleanup 1

PRMerger19 committed:
Merge pull request #35228 from neilpeterson/aks-government-patch

updated to AKS

CarlRabeler committed:
Merge branch 'master' into meta-2

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into wwi

barbkess committed:
fix loaderRC20 to loaderRC60

PRMerger20 committed:
Merge pull request #35253 from neilpeterson/aks-issue-5457

name updated for consistancy

david-stanford committed:
[ACI] change a couple things in the environment variable page

david-stanford committed:
update the wording

mtillman committed:
Added line for saving sign-in images

Commit:
Merge pull request #35254 from barbkess/wwi

acrolinx fixes

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5607

Removing text on use of barriers. Guidance is more complex and varies by distro / filesystem. articles/virtual-machines/linux/premium-storage#premium-storage-for-linux-vms.md should be canonical guidance.

GitHubber17 committed:
Merge pull request #35100 from eringreenlee/master

added service principal alert

mtillman committed:
Added some acrolinx fixes

rjagiewich committed:
Merge pull request #35149 from kgremban/mar12_5346

Updating module comp article for issue #5346

david-stanford committed:
add a comma after the dependent clause

rjagiewich committed:
Merge pull request #35244 from douglaslMS/adf-tutorial-toc

Moved new article in TOC per PM request.

PRMerger16 committed:
Merge pull request #35190 from msjuergent/hanaopsguide

wa_doc7

adegeo committed:
Adjusted limits per email thread

SnehaGunda committed:
Fixing PR review comments

PRMerger17 committed:
Merge pull request #35259 from david-stanford/small-fix

[ACI] change a couple things in the environment variable page

mtillman committed:
more acrolinx fixes

Commit:
tocfix04

Commit:
tocfix04

Commit:
Merge pull request #35260 from mtillman/patch-3

Added line for saving sign-in images

gsacavdm committed:
Correct acs-engine version in gov-k8

Commit:
Merge pull request #35267 from Thraka/service-limit-fix

Adjusted limits per email thread

rwike77 committed:
embedded video

rwike77 committed:
updated text

PRMerger14 committed:
Merge pull request #35262 from iainfoulds/patch-18

[VM] Fixes MicrosoftDocs/azure-docs#5607

PRMerger15 committed:
Merge pull request #35115 from pharring/master

Change MinidumpUploader to SnapshotUploader

Commit:
quickfix78

PRMerger17 committed:
Merge pull request #35271 from rwike77/videos

embedded video

rwike77 committed:
embedded video

tamram committed:
Update azure-storage-limits.md

PRMerger18 committed:
Merge pull request #35275 from MarkusVi/quickfix78

quickfix78

Commit:
Adding delete recommendation

barbkess committed:
fix typos

j-martens committed:
AML: Updating metadata for tutorial 3

Commit:
deltopic05

Commit:
Merge pull request #35276 from rwike77/videos

embedded video

Commit:
Merge pull request #35277 from MicrosoftDocs/tamram-patch-1

Update azure-storage-limits.md per issue 5651

Commit:
Update stream-analytics-use-reference-data.md

Commit:
Merge pull request #35279 from barbkess/wwi

fix typos

CawaMS committed:
adding more scenarios

vladvino committed:
Updated C# version

PRMerger17 committed:
Merge pull request #35278 from JSeb225/patch-25

Adding delete recommendation

asmalser-msft committed:
Added info about attribute customization

Commit:
Merge pull request #35285 from vladvino/patch-8

Updated C# version

Lisaco88 committed:
Merge pull request #35170 from gsacavdm/patch-5

Add azure-gov K8 documentation

asmalser-msft committed:
Fixed image

asmalser-msft committed:
Fixed image

Lisaco88 committed:
Merge pull request #35212 from SnehaGunda/CL

Adding VS tools edge topic

Lisaco88 committed:
Merge pull request #35245 from kgremban/mar13_5430

Add clarifying info about ACR

Lisaco88 committed:
Merge pull request #35250 from CarlRabeler/meta-2

[sql db] meta data cleanup 2

PRMerger17 committed:
Merge pull request #35286 from christiankuhtz/patch-71

tweak

christiankuhtz committed:
Tweak

Lisaco88 committed:
Merge pull request #35269 from MarkusVi/tocfix04

tocfix04

Lisaco88 committed:
Merge pull request #35280 from MarkusVi/deltopic05

deltopic05

Commit:
Merge pull request #35292 from christiankuhtz/patch-77

Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
Tweak

christiankuhtz committed:
tweak

PRMerger12 committed:
Merge pull request #35293 from christiankuhtz/patch-78

Tweak

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5701

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5701

PRMerger20 committed:
Merge pull request #35296 from MohitGargMSFT/patch-8

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5701

asmalser-msft committed:
Grammar fixes

SnehaGunda committed:
removing the extra line

Sogup committed:
Support for 4TB disk released

Commit:
Merge pull request #35297 from SnehaGunda/CL

removing the extra line

Sogup committed:
4TB disk support released

PRMerger18 committed:
Merge pull request #35301 from Sogup/patch-38

4TB disk support released

dominicbetts committed:
Remove "underscore"

Commit:
Merge pull request #35305 from dominicbetts/remove-underscore

Remove "underscore"

asmalser-msft committed:
Style edits

rastala committed:
Update text-analytics-module-tutorial.md

mimig1 committed:
Cosmos DB: Added line for setting coll throughput

asmalser-msft committed:
Typo

mimig1 committed:
Cosmos DB: Added info about using WindowsAzure.Storage

PRMerger17 committed:
Merge pull request #35307 from mimig1/patch-222

Cosmos DB: Added line for setting coll throughput

mumian committed:
[HDI] update the PowerBI connectors information

dominicbetts committed:
Update C code for firmware update

itechedit committed:
Merge pull request #35255 from CarlRabeler/meta-3

[sql db] metadata clean-up 3

mumian committed:
fix links

PRMerger16 committed:
Merge pull request #35282 from j-martens/master

AML: Updating metadata for tutorial 3

PRMerger16 committed:
Merge pull request #35308 from mimig1/patch-223

Cosmos DB: Added info about using WindowsAzure.Storage

PRMerger17 committed:
Merge pull request #35309 from dominicbetts/suite-c-updates

Update C code for firmware update

PRMerger17 committed:
Merge pull request #35310 from mumian/0314-powerbi

[HDI] update the PowerBI connectors information

rjagiewich committed:
Merge pull request #35300 from Sogup/patch-37

Support for 4TB disk released

Commit:
Merge pull request #35306 from rastala/patch-1

Update text-analytics-module-tutorial.md

Lisaco88 committed:
Merge pull request #35288 from asmalser-msft/master

Added info about attribute customization

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
minor update

Lisaco88 committed:
Merge pull request #32897 from v-nagta/jiraconfluencev1

USER STORY 1178697 SaaS App Tutorial: Confluence SAML SSO by Microsof…

cherylmc committed:
Update vpn-gateway-download-vpndevicescript.md

PRMerger18 committed:
Merge pull request #35284 from CawaMS/master

adding more scenarios for custom containers

nitinme committed:
fixing a file link

v-alje committed:
Merge pull request #35318 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #35317 from nitinme/nitinme-databricks-updates

Minor update

Ja-Dunn committed:
Merge pull request #34822 from ShawnJackson/virtual-machine-scale-sets-upgrade-scale-set

edit pass: virtual-machine-scale-sets-upgrade-scale-set

Commit:
Merge pull request #35324 from cherylmc/patch-1

Update vpn-gateway-download-vpndevicescript.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
